plugins/janus_videoroom.c

```diff
@@ -2699,7 +2699,18 @@ static void janus_videoroom_recorder_create(janus_videoroom_publisher *participa
 		}
 	}
 }
-
+static void janus_videoroom_recorder_pause(janus_videoroom_publisher *participant, gboolean paused) {
```

Comment:
See below for the semantic names. In this case, to keep things compact, the name could be changed to `janus_videoroom_recorder_status` or something like this, in order to keep the boolean flag.